lib/screens/monster/monster_info_subtab.dart

```diff
@@ -529,13 +538,32 @@ class MonsterAssistStatTable extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     var m = _data.monster;
-    // TODO: account for stat boosts
+    var a = _data.awakenings;
     var hpMax = m.hpMax * .1;
     var atkMax = m.atkMax * .05;
     var rcvMax = m.rcvMax * .15;
     var hp297Max = hpMax + 99 * 10 * .1;
     var atk297Max = atkMax + 99 * 5 * .05;
     var rcv297Max = rcvMax + 99 * 3 * .15;
+    var isEquip = false;
+    
+    // Only add stat changes if assist type
+    if (a[a.length - 1].awokenSkill.nameNa == "Awoken Assist") {
```

Comment:
Would you mind changing this to use `a.any((as) => as.awokenSkill.id == X)` where X is the awakening id from the json file?

Eventually I'll add an enum for that but the ID is fine for now.

Reply:
Good idea, I just pushed the change.

Reply:
Thanks, lgtm